From 1558c2678c0c85fbfce2db56e620ae3d321e5112 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Thu, 17 Dec 2015 14:25:28 +0200
Subject: [PATCH 24/66] perf: Add compilation of perf kernel utility

	This requires compiling the kernel, although the Kernel image will not be used.

Change-Id: Ia71ea11c993731eddb9cb9b04c8b094dc6b6800a
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26089
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
---
 configs/mvebu_armv8_le_defconfig | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/configs/mvebu_armv8_le_defconfig b/configs/mvebu_armv8_le_defconfig
index f7b9ed8..64ce950 100644
--- a/configs/mvebu_armv8_le_defconfig
+++ b/configs/mvebu_armv8_le_defconfig
@@ -3,6 +3,11 @@ BR2_TOOLCHAIN_EXTERNAL=y
 BR2_MVEBU_RELEASE_NAME="2016_T1.0.eng_drop_v1"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files/"
+BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.1.6"
+BR2_LINUX_KERNEL_DEFCONFIG="defconfig"
+BR2_LINUX_KERNEL_TOOL_PERF=y
 BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
 BR2_PACKAGE_BZIP2=y
 BR2_PACKAGE_GZIP=y
@@ -15,7 +20,6 @@ BR2_PACKAGE_STRACE=y
 BR2_PACKAGE_STRESS=y
 BR2_PACKAGE_WHETSTONE=y
 BR2_PACKAGE_GREP=y
-BR2_PACKAGE_TAR=y
 BR2_PACKAGE_BTRFS_PROGS=y
 BR2_PACKAGE_CRAMFS=y
 BR2_PACKAGE_E2FSPROGS_DEBUGFS=y
@@ -24,7 +28,6 @@ BR2_PACKAGE_DEVMEM2=y
 BR2_PACKAGE_HDPARM=y
 BR2_PACKAGE_I2C_TOOLS=y
 BR2_PACKAGE_IOSTAT=y
-BR2_PACKAGE_KVMTOOL=y
 BR2_PACKAGE_LM_SENSORS=y
 BR2_PACKAGE_LSHW=y
 BR2_PACKAGE_MDADM=y
@@ -47,7 +50,6 @@ BR2_PACKAGE_LIGHTTPD=y
 BR2_PACKAGE_NCFTP=y
 BR2_PACKAGE_NETCAT=y
 BR2_PACKAGE_RSYNC=y
-BR2_PACKAGE_SAMBA=y
 BR2_PACKAGE_TCPDUMP=y
 BR2_PACKAGE_VSFTPD=y
 BR2_PACKAGE_WGET=y
@@ -58,6 +60,8 @@ BR2_PACKAGE_TIME=y
 BR2_PACKAGE_WHICH=y
 BR2_PACKAGE_FTOP=y
 BR2_PACKAGE_HTOP=y
+BR2_PACKAGE_KVMTOOL=y
+BR2_PACKAGE_TAR=y
 BR2_PACKAGE_LESS=y
 BR2_PACKAGE_VIM=y
 BR2_TARGET_ROOTFS_TAR_GZIP=y
-- 
1.9.1

